auth/api/iam/api.go

```diff
@@ -316,22 +297,12 @@ func (r Wrapper) HandleAuthorizeRequest(ctx context.Context, request HandleAutho
 	// Workaround: deepmap codegen doesn't support dynamic query parameters.
 	//             See https://github.com/deepmap/oapi-codegen/issues/1129
 	httpRequest := ctx.Value(httpRequestContextKey).(*http.Request)
-	params := parseQueryParams(httpRequest.URL.Query())
-	clientId := params.get(oauth.ClientIDParam)
+	qParams := httpRequest.URL.Query()
```

Comment:
qParams and params are now somewhat ambiguous, maybe `queryParams` and `requestObject`